src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        try {
+            return movieDao.add(movie);
+        } catch (Exception e) {
+            throw new DataProcessingException("Error while adding movie: " + movie, e);
+        }
```

Comment:
you have try/catch blocks on dao level, there is no need to duplicate it in service